package.json

```diff
@@ -81,17 +81,24 @@
     "github": "^1.1.1",
     "glob": "^7.1.3",
     "json-typescript": "^1.1.0",
-    "lerna": "^3.13.2",
+    "lerna": "^3.13.4",
     "loader.js": "^4.7.0",
     "mocha": "^6.1.2",
     "mocha-only-detector": "1.0.1",
     "prettier": "^1.15.3",
     "rimraf": "^2.6.2",
     "rsvp": "^4.8.4",
-    "typescript": "~3.4.3"
+    "typescript": "~3.4.3",
+    "chalk": "^2.4.1",
+    "debug": "^4.1.1",
+    "semver": "^6.0.0"
   },
   "bin": {
     "test-partner-project": "./bin/test-external-partner-project.js",
-    "prepare-local-packages": "./bin/packages-for-commit.js"
+    "prepare-local-packages": "./bin/packages-for-commit.js",
+    "publish": "./bin/publish.js"
+  },
+  "dependencies": {
+    "command-line-args": "^5.1.1"
```

Comment:
Should this be a dev-dependency instead?

Reply:
likely, I intended it to be but it snuck up here. I don't think it matters since this is root but better to be safe.